pkg/clientutil/predicates.go

```diff
@@ -53,6 +53,16 @@ func PredicateClusterByAccessMode(accessMode greenhousev1alpha1.ClusterAccessMod
 	})
 }
 
+func PredicateClusterIsReady() predicate.Predicate {
+	return predicate.NewPredicateFuncs(func(o client.Object) bool {
+		cluster, ok := o.(*greenhousev1alpha1.Cluster)
+		if ok && (len(cluster.Status.Conditions) == 0 || !cluster.Status.IsReadyTrue()) {
+			return true
```

Comment:
this seems to be the false way around? expect it to return `true` if the cluster is ready, right?

Reply:
We could even refactor the check into it's own utility function then also used in other `ClusterReady` checks, right?

Reply:
I did not find any other `ClusterReady` checks. Let's implement such utilities once we have a concrete need.